internal/states/remote/remote_grpc_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package remote
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/internal/providers"
+	testing_provider "github.com/hashicorp/terraform/internal/providers/testing"
+)
+
+// Testing grpcClient's Delete method.
+// This method is needed to implement the remote.Client interface, but
+// this is not invoked by the remote state manager (remote.State) that
+// wil contain the client.
+//
+// In future we should remove the need for a Delete method in
+// remote.Client, but for now it is implemented and tested.
+func Test_grpcClient_Delete(t *testing.T) {
+	typeName := "my-workspace"
+	stateId := "foobar"
```

Comment:
nitpick - could we swap out the two test strings? It just makes it a bit confusing when the state store is called `my-workspace` - something that one would expect state ID (workspace) to be called.